api/proto/metrics/metrics.proto

```diff
@@ -126,6 +126,11 @@ message deviceMetric {
   memoryMetric systemServicesMemoryMB = 11;  // In MB
   logMetric log = 12;
   repeated CipherMetric cipher = 13;  // Object decryption
+  AclMetric acl = 14;
```

Comment:
Is all we need a global count for the device and not per network-instance or per app-instance attachment to a network instance? If this is sufficient let's go for it.

Reply:
Yes, that what the integration test code does currently. Get a global count of rules added to filter, raw tables before and after the tests. Test checks if the acl count increased after adding app/apps.